### DISCUSSION TITLE
Need Documentation for flet_core module

### DISCUSSION THREAD

**Discussion by gunaNeelamegam** [flet-dev/flet#4148] [Q&A]
### Duplicate Check

- [X] I have searched the [opened issues](https://github.com/flet-dev/flet/issues) and there are no duplicates

### Describe the requested feature

Need documentation for flet core module api's for integrate with custom dart package

**Comment by ndonkoHenri** [comment]
See this: https://flet.dev/docs/extend/integrating-existing-flutter-packages

**Reply by gunaNeelamegam** [reply]
I known how to integrate with 3rd party package's or custom package. But, I need to known about the api doc for trigger the event and receive the event.

* Now, flet Control class behaves like observer pattern on_event_name.

But, we read the flet_core source code only we can able to understand what is going on inside.
I need to understand more indepth.

**Reply by ndonkoHenri** [reply]
You might find my Flet-Contribution series helpful: https://youtube.com/playlist?list=PLHYEoLmkidc1YrgREPrJvhr_JEv00EFP8

**Reply by gunaNeelamegam** [reply]
Thankyou @ndonkoHenri , It's helpful
